buf/registry/module/v1beta1/commit.proto

```diff
@@ -48,17 +53,31 @@ message Commit {
     (buf.validate.field).required = true,
     (buf.validate.field).string.uuid = true
   ];
-  // The digest of the Commit's contents.
+  // The Digest of the Commit's contents.
```

Comment:
The term `content` here was confusing to me because in this context it includes metadata about dependencies to other commits but in another context (DownloadService) it refers to files only.

Reply:
Nevermind, I misread that Content includes deps. Ignore this.